### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by JsseL**
Get rid of few defects that have lurked into the repo.

### CODE REVIEW

File: cmsplugin_articles_ai/apps.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from django.apps import AppConfig
+
+
+class RockNRollConfig(AppConfig):
```
Comment (erej) [inline cmsplugin_articles_ai/apps.py]: ROCK ON! 🎸 🎸 🥁  :D Even though I like this one, please change this.

File: cmsplugin_articles_ai/__init__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 #      .--.
 #       /    \
 #      ## a  a       _
-#      (   '._)     |_|
+#      (   '._)    c|_|
```
Comment (erej) [inline cmsplugin_articles_ai/__init__.py]: I'd also might remove this butler. Or maybe if we enforce these for all the plugins then this butler can live on

**Review by erej — CHANGES_REQUESTED** [review]
Naming mismatch 😮

**Comment by erej** [comment]
Travis settings probably needs another look too

File: cmsplugin_articles_ai/apps.py (hunk 1 above)
Comment (JsseL) [reply cmsplugin_articles_ai/apps.py]: Ups, copy pasta :japanese_ogre:

File: cmsplugin_articles_ai/__init__.py (hunk 2 above)
Comment (JsseL) [reply cmsplugin_articles_ai/__init__.py]: His bottle now  has a handle.. I think it's worth keeping..

Comment (JsseL) [reply cmsplugin_articles_ai/__init__.py]: Ok, removed it :smile:

Comment (erej) [reply cmsplugin_articles_ai/__init__.py]: Fair enough, lets keep him :)

**Review by erej — APPROVED** [review]

**Pull request merged**